README.md

```diff
@@ -92,51 +46,14 @@ Run a Kubernetes proxy to allow the dm client to connect to the cluster:
 kubectl proxy --port=8001 --namespace=dm &
 ```
 
-### Deploy an app
```

Comment:
Seems like a good idea to keep some simple "getting started" examples here that only take 1 or 2 lines to do something interesting. I'm fine with changing the ones we had, but not with removing them.